lzy/lzy-service/src/main/java/ai/lzy/service/LzyService.java

```diff
@@ -751,17 +751,22 @@ public void getOrCreateDefaultStorage(GetOrCreateDefaultStorageRequest request,
         final String userId = currentSubject().id();
         final String bucketName = StorageUtils.createInternalBucketName(userId);
 
+        var idempotencyKey = IdempotencyUtils.getIdempotencyKey(request);
+
+        var client = idempotencyKey == null
+            ? storagesGrpcClient()
+            : withIdempotencyKey(storagesGrpcClient(), idempotencyKey.token());
```

Comment:
lets add some suffix to original idempt key, e.g. "_create_storage". This way the general idemp key will not be occupied by specific operation